src/runtime_src/tools/scripts/service_bundle.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2019-2021 Xilinx, Inc. All rights reserved.
+#
+# This script collects debugging logs for XRT managed sub-systems
+
+TIME=`date +%F-%T`
+ID=`id -un`
+FULL_CMD=${BASH_SOURCE[0]}
+BASE_CMD=`basename "$0"`
+
+progress() {
```

Comment:
To help future developers, having a comment here describing what each argument is would be helpful, for example:
`# $1 - Index; $2 - Total; $3 - Function`

Reply:
fixed